frontend/src/metabase/dashboard/containers/CopyDashboardForm.tsx

```diff
@@ -47,7 +46,7 @@ export interface CopyDashboardFormProperties {
 }
 
 export interface CopyDashboardFormProps {
-  onSubmit?: (values: CopyDashboardFormProperties) => Dashboard;
+  onSubmit: (values: CopyDashboardFormProperties) => Promise<Dashboard>;
```

Comment:
This type was wrong.
1. We always pass `onSubmit`
2. the return type is a promise
    https://github.com/metabase/metabase/blob/fd8a4fe8a2e8f7f0d8e3d4d93872e4bb8224dba0/frontend/src/metabase/entities/containers/EntityCopyModal.tsx#L15